lib/network/responses/image_response.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'image_response.g.dart';
+
+@JsonSerializable()
+class MyImage {
+  int? id;
```

Comment:
prefer final values
[https://dart.academy/immutable-data-patterns-in-dart-and-flutter/](url)

Reply:
change Immutable Data